src/screens/AdminPortal/OrganizationFundCampaign/modal/CampaignModal.spec.tsx

```
@@ -39,6 +44,83 @@ vi.mock('@mui/x-date-pickers/DateTimePicker', async () => {
+    onHide,
+    title,
+  }: {
+    children: React.ReactNode;
+    show: boolean;
+    onHide: () => void;
+    title: string;
+  }) =>
+    show ? (
+      <div data-testid="base-modal">
+        <h2 data-testid="modal-title">{title}</h2>
+        <button data-testid="modal-close" onClick={onHide}>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)